Automattic/jetpack
# VideoPress: Add Pagination component

Closes #25743

#### Changes proposed in this Pull Request:

* Add component Pagination to VideoPress Package

#### Other information:


#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Run `cd projects/js-packages/storybook && pnpm run storybook:dev`
* Confirm that the default story of `Packages/VideoPress/Pagination` component works as expected

![Screenshot_2022-08-26_14-08-29](https://user-images.githubusercontent.com/8486249/186959435-86386e4e-5baa-482f-b0c9-b3d73fc1e07b.png)
![Screenshot_2022-08-26_14-07-54](https://user-images.githubusercontent.com/8486249/186959457-cbcc0991-3aaf-4236-9218-a5b3f23b5bc4.png)
![Screenshot_2022-08-26_14-08-44](https://user-images.githubusercontent.com/8486249/186959468-eeeb5942-37e0-4304-842d-d139237147c6.png)
![Screenshot_2022-08-26_14-08-55](https://user-images.githubusercontent.com/8486249/186959478-7fc7ccdd-32b8-445a-b8c1-dea10ccf278e.png)
